org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/markers/MarkerUtils.java

```diff
@@ -53,6 +55,10 @@ public final class MarkerUtils {
 
   public static final Set<String> SONARLINT_PRIMARY_MARKER_IDS = Set.of(
     SonarLintCorePlugin.MARKER_ON_THE_FLY_ID, SonarLintCorePlugin.MARKER_REPORT_ID, SonarLintCorePlugin.MARKER_TAINT_ID);
+  
+  public static final String SONAR_MARKER_MODE_STANDALONE = "standalone";
```

Comment:
I wonder if an enum would not be more appropriate here

Reply:
Done